libraries/AP_InertialNav/AP_InertialNav_NavEKF.cpp

```diff
@@ -78,20 +78,6 @@ float AP_InertialNav_NavEKF::get_altitude() const
 }
 
 /**
- * getHgtAboveGnd - get latest height above ground level estimate in cm and a validity flag
```

Comment:
Instead of removing parts of inertial-nav one by one, perhaps just a single commit at the end to completely remove the library would be best.